frontend/src/vars.scss

```diff
@@ -151,6 +150,7 @@ $light-grey: #f2f2f2;
 
 $default_spacing: 1rem;
 $radius: 4px;
+$opacity: 0.6;
```

Comment:
Hm, this is super vague, I suggest renaming to `$opacity_disabled`/`--opacity-disabled`